src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+
+interface Props {
+  hideErrorMessage: () => void,
+  errorMessage: string | null,
+}
+
+export const ErrorNotification: React.FC<Props> = (props) => {
+  const { hideErrorMessage, errorMessage } = props;
+  const [isVisible, setIsVisible] = useState(false);
+
+  useEffect(() => {
+    if (errorMessage) {
+      setIsVisible(true);
+
+      const timeout = setTimeout(() => {
+        hideErrorMessage();
+      }, 3000);
+
+      return () => clearTimeout(timeout);
+    }
+
+    return () => {};
```

Comment:
use a [guard clause](https://learningactors.com/javascript-guard-clauses-how-you-can-refactor-conditional-logic/) instead of doing this empty return later 
```suggestion
    if (!errorMessage) {
    return;
    }
    
      setIsVisible(true);

      const timeout = setTimeout(() => {
        hideErrorMessage();
      }, 3000);

      return () => clearTimeout(timeout);
```